### ISSUE TITLE
Not working in UE 5.1

### ISSUE THREAD

**Issue by Hannessa** [ncsoft/Unreal.js#329]
Hi,

I tried to install Unreal.js in a brand new UE 5.1 project, following the instructions in the readme. However, I just get the following error message when I try to rebuild it:

![image](https://user-images.githubusercontent.com/1542042/211943385-7f9986b7-1ae0-4458-87dd-7aa38511bd8a.png)

![image](https://user-images.githubusercontent.com/1542042/211943885-6d7fb1c3-ec21-4ee1-89e1-37f05b1f5e6b.png)

![image](https://user-images.githubusercontent.com/1542042/211943697-fce4f468-4682-4996-a3d4-6b96bfd7daa3.png)

**Comment by Hannessa** [comment]
Here's a list of the compile errors I get:

![image](https://user-images.githubusercontent.com/1542042/216344816-7af6d033-1d01-415f-a717-a123dace40aa.png)

**Comment by Hannessa** [comment]
I managed to solve half of them by just adding 

`#include "EditorStyleSet.h"
`

at the top of SJavascriptGraphEdNode.cpp

**Comment by dustout** [comment]
I ran into the same issue and ended up rolling back to UE5, am happy to know I am not the only one having this problem

**Comment by Hannessa** [comment]
Yeah, same here, good to know! I also rolled back, but I do want to start switching to UE5.1 soon thanks to the much improved foliage rendering.

I also added:
#include "EditorStyleSet.h" 
to JavascriptEditorTab.cpp (on row 5 below #if WITH_EDITOR)

and I'm now down to 17 errors. So that gives me some hope that it might mostly be easy fixes.

![image](https://user-images.githubusercontent.com/1542042/216348501-33778d2a-ed93-4dfb-a9d4-ff977fffa7c0.png)

Not quite sure why this didn't generate errors before, and I can't find any info about this EditorStyleSet.h in the [UE 5.1 release notes](https://docs.unrealengine.com/5.1/en-US/unreal-engine-5.1-release-notes/). But maybe it's just a newer compiler or something that is less forgiving.

**Comment by dustout** [comment]
I'm cheering for you! I'll take a deeper look in the evening and see if I can figure it out as well, fingers crossed that one of us can solve this problem.

**Comment by getnamo** [comment]
This changelist may be helpful for updating Unreal.js to 5.1: https://github.com/getnamo/UnrealJs/commit/3836ff244bc3c48aecc7f1a7475433f15c66bc03.

and https://github.com/getnamo/UnrealJs/commit/733c7638b99fc6a7977a1c7dcbba28eead00e45c for re-enabling logging.

There are still some deprecation warnings on compile, but it does run.

**Comment by getnamo** [comment]
Made minor fixes and made a fork release (win64) for 5.1: https://github.com/getnamo/UnrealJs/releases/tag/v0.7.0

**Comment by dochigun-nc** [comment]
UE 5.1 supported version has been pushed.

**Comment by Johnly1986** [comment]
@dochigun-nc The new version can't run the example normally, and input "GWorld" in the JavaScript console prompts an undefined error.
